### ISSUE TITLE
Support for private docker registry?

### ISSUE THREAD

**Issue by alu-ixxus** [spotify/dockerfile-maven#8]
Hi,

I would like to know if this plugin officially supports pulling from a private docker registry? 

I get authentication issue when I try to pull from an image from a private docker registry.

Thanks

**Comment by dkanejs** [comment]
Any clarification on this? I also get an auth error.

**Comment by mattnworb** [comment]
FYI we are planning improvements in https://github.com/spotify/docker-client/issues/740 that should make it easier for this project to support authentication once 740 is complete.

**Comment by jpwijbenga** [comment]
I'd like to have some more info in the readme on whether the plugin supports authentication for a private registry.
I tried to push to a private Gitlab CI registry using this plugin, but got a 'Forbidden' response from the registry. Probably, because I didn't do a 'docker login', I think.
It would be nice to have some documentation in the readme (also add if it's not supported yet). Thanks!

**Comment by mattnworb** [comment]
v1.3.0 of the plugin adds support for reading the authentication info from your ~/.dockercfg or ~/.docker/config.json file, see the README for more.
